Elasticsearch - fixing fetch with timestamps (#6941)
demisto/content: Packs/Elasticsearch/Integrations/Elasticsearch_v2/CHANGELOG.md

## [Unreleased]

> Fixed an issue where ***fetch-incident*** would raise an error in case the *Time field type* integration parameter is set to "Timestamp-Millisecond" or "Timestamp-Seconds".

Reviewer: ```suggestion
Fixed an issue where ***fetch-incident*** would raise an error when the *Time field type* integration parameter is set to "Timestamp-Millisecond" or "Timestamp-Seconds".
```
